department-of-veterans-affairs/va.gov-cms
# VACMS-2867: VA Benefit Hubs Field Content Proofing

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/2867

## Testing done


## Screenshots


## QA steps

As an editor
- [ ] Log into preview environment. (This can be found in devshop) 

- Navigate to the Benefit Contact Page Content Type Manage Display Page:
 - [ ] Navigate to: `/admin/structure/types/manage/landing_page/display/teaser`
 - Confirm The following fields exist:
  - [x] Hub Label | Label: Hidden | Format: Plain text | Linked to the content
  - [x] Hub teaser text | Label: Hidden | Format: Plain text

### Add content for the following content types to confirm the teaser view mode of the VA Benefit Hubs field

## checklist 

- [ ] Navigate to `node/add/checklist`
  - [ ] Complete all required fields and be sure to add at least an additional VA Benefit Hubs entity reference (_Should already default to VA health care_) .
  - Save form.
  - Focus on the VA Benefit Hubs section and verify the following:
    - [x] The Hub Label is a link to the Hub view page.
    - [x] The Hub teaser text is displayed below the Hub Label as plain text.



## faq_multiple_q_a

- [ ] Navigate to `node/add/faq_multiple_q_a`
  - [ ] Complete all required fields and be sure to add at least an additional VA Benefit Hubs entity reference (_Should already default to VA health care_) .
  - Save form.
  - Focus on the VA Benefit Hubs section and verify the following:
    - [x] The Hub Label is a link to the Hub view page.
    - [x] The Hub teaser text is displayed below the Hub Label as plain text.



## step_by_step

- [ ] Navigate to `node/add/step_by_step`
  - [ ] Complete all required fields and be sure to add at least an additional VA Benefit Hubs entity reference (_Should already default to VA health care_) .
  - Save form.
  - Focus on the VA Benefit Hubs section and verify the following:
    - [x] The Hub Label is a link to the Hub view page.
    - [x] The Hub teaser text is displayed below the Hub Label as plain text.


## media_list_images

- [ ] Navigate to `node/add/media_list_images`
  - [ ] Complete all required fields and be sure to add at least an additional VA Benefit Hubs entity reference (_Should already default to VA health care_) .
  - Save form.
  - Focus on the VA Benefit Hubs section and verify the following:
    - [x] The Hub Label is a link to the Hub view page.
    - [x] The Hub teaser text is displayed below the Hub Label as plain text.



## media_list_videos

- [ ] Navigate to `node/add/media_list_videos`
  - [ ] Complete all required fields and be sure to add at least an additional VA Benefit Hubs entity reference (_Should already default to VA health care_) .
  - Save form.
  - Focus on the VA Benefit Hubs section and verify the following:
    - [x] The Hub Label is a link to the Hub view page.
    - [x] The Hub teaser text is displayed below the Hub Label as plain text.




## q_a

- [ ] Navigate to `node/add/manage/q_a`
  - [ ] Complete all required fields and be sure to add at least an additional VA Benefit Hubs entity reference (_Should already default to VA health care_) .
  - Save form.
  - Focus on the VA Benefit Hubs section and verify the following:
    - [x] The Hub Label is a link to the Hub view page.
    - [ ] The Hub teaser text is displayed below the Hub Label as plain text.





## support_resources_detail_page

- [ ] Navigate to `node/add/support_resources_detail_page`
  - [ ] Complete all required fields and be sure to add at least an additional VA Benefit Hubs entity reference (_Should already default to VA health care_) .
  - Save form.
  - Focus on the VA Benefit Hubs section and verify the following:
    - [x] The Hub Label is a link to the Hub view page.
    - [x] The Hub teaser text is displayed below the Hub Label as plain text.

## Definition of Done
